ImgShareData/ImgurEndpoints.cs

```diff
@@ -153,50 +153,20 @@ static public String GallerySearch(GallerySort sort, GallerySearchWindow window,
             }
 
         }
-        #endregion
-
-        static public Dictionary<ImageEndpointParameters, String> ImageEndpointParameterLookup = new Dictionary<ImageEndpointParameters, string>()
-        {
-            {ImageEndpointParameters.image, "image"},
-            {ImageEndpointParameters.album, "album"},
-            {ImageEndpointParameters.description, "description"},
-            {ImageEndpointParameters.title, "title"}
-        };
-    }
-
-    public enum GallerySection
-    {
-        hot,
-        top,
-        user
-    };
-
-    public enum GallerySort
-    {
-        viral,
-        time
-    };
-
-    public enum ImageEndpointParameters
-    {
-        image,
-        title,
-        description,
-        album
-    };
-
-    public enum ImgurEndpointUseType
-    {
-        free,
-        paid
-    };
-
-    public enum GallerySearchWindow
-    {
-        day,
-        week,
-        month,
-        year,
-        all
-    };
-}
+        #endregion
+
+        static public Dictionary<ImgurParameters, String> ImageEndpointParameterLookup = new Dictionary<ImgurParameters, string>()

```

Comment:
At the original time of inception, I didn't have a better idea for how to manage the different endpoint parameter strings.  Womp womp?  Bad enough to change it?